onnx_tf/backend.py

```diff
@@ -180,11 +180,11 @@ def get_padding_as_op(cls, x, pads):
 
   @classmethod
   def _explicit_broadcast(cls, tensor, broadcast_dim=1, total_num_dim=4):
-    if not isinstance(broadcast_dim, list):
-      broadcast_dim = [broadcast_dim]
-
+    if broadcast_dim < 0:
```

Comment:
I was hoping to support the case where broadcast_dim is a list.. something like broadcast_dim=[1, 2].

I wonder why the change to move away from such support?

Reply:
@tjingrant 
Maybe I missed sth. I think `broadcast_dim` is sth https://github.com/onnx/onnx/blob/master/docs/Operators.md#add metioned here.
So I think it is a start dim, and end dim is `broadcast_dim + len(tensor.shape) - 1`.
I also can do list `boardcast_dim`, because it is just `list(range(broadcast_dim, broadcast_dim + len(tensor.shape)))`.

What do you think?

Reply:
I think your interpretation makes more sense. Let's keep it this way then.